### PR TITLE
Make the contact information displayed by the Metaflow command configurable

### PR DESCRIPTION
**Pull request by romain-intel**
Also minor change to display the proper Metaflow version (including extension info) in the command line tool

### CODE REVIEW

File: metaflow/cmd/main_cli.py [hunk 1]
```diff
@@ -80,24 +81,18 @@ def start(ctx):
...
-        echo(" - Read the documentation")
-
-        # metaflow chat
-        echo("http://chat.metaflow.org", fg="cyan", nl=False)
-        echo(" - Chat with us")
-
-        # metaflow help email
-        echo("help@metaflow.org", fg="cyan", nl=False)
-        echo("        - Get help by email\n")
+        lnk_sz = max(len(lnk) for lnk in CONTACT_INFO.values()) + 1
```
[the remark this answers is not in the document]
Comment (romain-intel) [reply metaflow/cmd/main_cli.py:91]: How so? The default is the exact same thing so unless the user overrides METAFLOW_CONTACT_VALUE, it shouldn't happen right?

[the remark this answers is not in the document]
Comment (savingoyal) [reply metaflow/cmd/main_cli.py:91]: only if you still want the user to ping `help@metaflow.org`. The tiny edge case is when the user should reach out to `help@foo.com` but `metaflow help` won't show that value unless metaflow has been configured properly.

[the remark this answers is not in the document]
Comment (romain-intel) [reply metaflow/cmd/main_cli.py:91]: We discussed this -- organizations can override using environment configs, things in ~/.metaflowconfig or the extension mechanism.

File: metaflow/metaflow_config.py [hunk 2]
```diff
@@ -193,7 +193,18 @@
 # Default container registry
 DEFAULT_CONTAINER_REGISTRY = from_conf("DEFAULT_CONTAINER_REGISTRY")
 
+###
+# Organization customizations
+###
 UI_URL = from_conf("UI_URL")
+CONTACT_INFO = from_conf(
```
[the remark this answers is not in the document]
Comment (romain-intel) [reply metaflow/metaflow_config.py:200]: I'll add a comment -- there are other dictionary config values that are not yet validated. I think it's a good call-out to actually validate everything (not just these) but that is a bigger endeavor than what I set out to do :).

**Review by saikonen — APPROVED** [review]

**Comment by romain-intel** [comment]
Thanks for the review!

**Pull request merged**